packages/fxa-auth-server/docs/api.md

```diff
@@ -789,6 +789,13 @@ Gets the status of an account.
   
   <!--end-query-param-get-accountstatus-uid-->
 
+##### Response Body
+
+* `authenticationMethods`: *isA.array().items(isA.string().required()).optional()*
+* `authenticatorAssuranceLevel`: *isA.number().min(0).optional()*
+* `exists`: *isA.boolean().required()*
+* `locale`: *isA.string().optional()*
```

Comment:
It's probably worth documenting that the authenticator-related properties are only returned when authed with a sessionToken (and ensuring that we have test coverage to this effect as well)

Reply:
>  (and ensuring that we have test coverage to this effect as well)

Ah, you already added such, awesome 👍